src/foam/dao/EasyDAO.js

```diff
@@ -166,6 +165,10 @@ if ( getDecorator() != null ) {
   delegate = (ProxyDAO) getDecorator();
 }
 
+if ( getValidated() ) {
+  delegate = getValidator();
```

Comment:
this doesn't make sense, the delegate is simply replaced, what happened to what the delegate previously delegated to?